pkg/packagekit/updates.jsx

```diff
@@ -551,7 +551,7 @@ class OsUpdates extends React.Component {
                         transactionPath, "org.freedesktop.DBus.Properties", "Get", [PK.transactionInterface, "Role"], {timeout: 5000}));
 
                     cockpit.all(promises)
-                            .done(roles => {
+                            .done((roles) => {
```

Comment:
There is no difference in these syntaxes. Parens around single-element tuples should generally be avoided.